sherpa-onnx/python/csrc/online-lm-config.cc

```diff
@@ -0,0 +1,23 @@
+// sherpa-onnx/python/csrc/online-lm-config.cc
+//
+// Copyright (c)  2023  Xiaomi Corporation
+
+#include "sherpa-onnx/python/csrc/online-lm-config.h"
+
+#include <string>
+
+#include "sherpa-onnx//csrc/online-lm-config.h"
+
+namespace sherpa_onnx {
+
+void PybindOnlineLMConfig(py::module *m) {
+  using PyClass = OnlineLMConfig;
+  py::class_<PyClass>(*m, "OnlineLMConfig")
+      .def(py::init<const std::string &, float>(), py::arg("model"),
+           py::arg("scale"))
```

Comment:
```suggestion
      .def(py::init<const std::string &, float>(), py::arg("model") = "",
           py::arg("scale") = 0.5f)
```

give them default values so that we don't need to set them explicitly in Python.